src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing` property is set to `border-box` only for the `body`. It's a common practice to apply this to all elements using the universal selector `*` to ensure consistent box-sizing across the entire page.